remoteworkitem/remoteworkitem_test.go

```diff
@@ -34,22 +34,14 @@ func provideRemoteData(dataUrl string) ([]byte, error) {
 	return responseData, nil
 }
 
-func provideRemoteGithubDataWithAssignee() ([]byte, error) {
+func provideRemoteGithubData() ([]byte, error) {
 	return provideRemoteData(GithubIssueWithAssignee)
 }
 
-func provideRemoteJiraDataWithAssignee() ([]byte, error) {
+func provideRemoteJiraData() ([]byte, error) {
 	return provideRemoteData(JiraIssueWithAssignee)
 }
 
-func provideRemoteGithubDataWithoutAssignee() ([]byte, error) {
```

Comment:
Please ignore this PR.
As I checked now `Files Changed`, these things are not changed by me.
Went wrong while rebase yesterday, I will have to close this PR and create new one with few changes. I used two repos (sbose's and alm's) for this branch and that caused much noise in rebase. Few changes are showing up as my commits which are not done by me. !! aaah rebase.

Closing this one, will create new, that will be much easy and quick. Or will try `git reset` may help.